order/models.py

```diff
@@ -25,6 +23,8 @@ def __str__(self):
 
 class Declaration(models.Model):
 
+    __status = None
```

Comment:
what is it?

Reply:
This is for keep the track of statuses

Reply:
This is to keep the track of status changes

Reply:
We talked about cache didnt we?

Reply:
You Just advised that i should have a variable to keep the status id

Reply:
Yes, but I did explain how it works right? this should in in __init__ function, so it is cached always when you fetch object from db